### PR TITLE
Fixes bluespace banana peels infinite loop.

### PR DESCRIPTION
**Pull request by xxalpha**
Probably fixes https://github.com/tgstation/-tg-station/issues/15926

Fixed bluespace banana icon.

### CODE REVIEW

**Comment by Core0verload** [comment]
#15944 has the same set of fixes.

File: code/modules/hydroponics/grown/banana.dm [hunk 1]
```diff
@@ -115,6 +116,8 @@
 /obj/item/weapon/grown/bananapeel/bluespace/Crossed(AM)
 	if(..())
 		var/teleport_radius = potency / 10
+		if(teleport_radius < 1) //not enough potency to teleport
+			return
 		do_teleport(AM, get_turf(AM), teleport_radius)
```
Comment (phil235) [inline code/modules/hydroponics/grown/banana.dm]: technically, it doesn't fix the infinite loop, it only makes it incredibly unlikely. You can still be teleported on the exact same tile and have 50% of being teleported again if the peel isn't deleted. Repeat endlessly.

Comment (xxalpha) [inline code/modules/hydroponics/grown/banana.dm]: That's supposed to be a feature or something.

Comment (Core0verload) [inline code/modules/hydroponics/grown/banana.dm]: Isn't teleport excluding the starting tile from possible target?

Comment (Core0verload) [inline code/modules/hydroponics/grown/banana.dm]: Isn't teleport excluding the starting tile from possible target?

Comment (Iamgoofball) [inline code/modules/hydroponics/grown/banana.dm]: this isn't a feature @xxalpha because if you get stuck in an endless teleport loop it kills the MC

**Pull request closed without merging**